guides/source/active_model_basics.md

```diff
@@ -122,68 +400,237 @@ class Person
   define_model_callbacks :update
 
   before_update :reset_me
+  after_update :finalize_me
+  around_update :log_me
 
   def update
     run_callbacks(:update) do
-      # This method is called when update is called on an object.
+      puts "update method called"
+      # This is the callback method executed when `update` is called on an object.
```

Comment:
Should these comments come before the code like throughout the rest of the guide? I think it would read better that way.
Also, sometimes comments begin lowercase and sometimes uppercase. We should choose one style and stick to it.